Client/Frontend/UIConstants.swift

```diff
@@ -50,8 +50,17 @@ public struct UIConstants {
     static let PrivateModeInputHighlightColor = UIColor(red: 120 / 255, green: 120 / 255, blue: 165 / 255, alpha: 1)
     static let PrivateModeAssistantToolbarBackgroundColor = UIColor(red: 89 / 255, green: 89 / 255, blue: 89 / 255, alpha: 1)
     static let PrivateModeToolbarTintColor = UIColor(red: 74 / 255, green: 74 / 255, blue: 74 / 255, alpha: 1)
-
-    static let ToolbarHeight: CGFloat = 46
+    static var ToolbarHeight: CGFloat {
```

Comment:
Hmm, on a second thought, it's probably not a good idea to have a `var` on a `UIConstants` file right?
It kinda is constant in a way that it will not change during the app lifetime (unless the insets are manually changed somewhere else), but doesn't feel right to have a `var` here, in the other hand, having a separated file and/or `struct` to have just this attribute might be overkill.

Reply:
The var looks fine to me being a run-time constant (vs a compile-time let constant).

Reply:
The thing with using toolbarHeight like this is that other parts of the app that show toolbars (like readermode/findinpage) use this height as well) In those cases we don't need the bottom insets. 

I think we should rename this to `BottomToolbarHeight` and make sure its only used for this specific case.

Reply:
Even the class `Toolbar` is used somewhere else where it is not on the bottom of the screen. 
I think what we want to do here is instead build the insets into `TabToolbar` which is a specific subclass of the `Toolbar` which appears only at the bottom of the screen. And then also build the safeInsets for `TrayToolbar`. That way the insets don't mess with anything else.